src/common/transformations/include/transformations/op_conversions/convert_convolution.hpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/matcher_pass.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+namespace pass {
+
+class TRANSFORMATIONS_API ConvertConvolution;
+
+}  // namespace pass
+}  // namespace ov
+
```

Comment:
please add description for this transofrmation

Reply:
Where it is better to add such description? This is transformation from convolution of the particular opset to the convolution of the internal opset